server/server.js

```diff
@@ -269,10 +270,18 @@ app.get('/genotet', function(req, res) {
 
     // Bed data queries
     case 'bed':
-      var file = req.query.file;
+      var file = req.query.fileName;
       var chr = req.query.chr;
-      var dir = bedPath + file + '_chr/' + file + '_' + chr;
-      data = bed.readBed(dir, req.query.xl, req.query.xr);
+      var dir = bedPath + file + '_chr/' + file + '_chr' + chr;
```

Comment:
In our assumption, fileName is unique but bedName could be the same for several different bed files. I have some issues about sync refactor so I will use motifs() for future, this is the old version. In upload function, I have separated one *.bed file into several files according to the chromosomes, and build a folder for them. bedPath is the folder to the bed files, the folder for one bed file is fileName_chr/, and for each chromosome, the file is fileName_chr1 for chromosome1.